ush/create_model_configure_file.py

```diff
@@ -90,6 +90,34 @@ def create_model_configure_file(
     # parameters and a set of grid parameters.  The latter depends on the type
     # (coordinate system) of the grid that the write-component will be using.
     #
+    # First, set the jinja template variables needed when using quilting (i.e. when
+    # outputting on the write-component grid) to default values.  If this is not
+    # done, the run_fcst task will fail with a "variables are not provided" message.
+    #
+    settings.update(
```

Comment:
This would be better as an "else" statement under the following "if QUILTING" block.

Reply:
@mkavulich Moved into an else-statement.